spec/spec_helper.rb

```diff
@@ -1,14 +1,7 @@
-require 'rubygems'
 require 'rspec/autorun'
-require "active_record"
-require 'active_support'
+require 'rails/all'
 require 'sqlite3'
-
-$LOAD_PATH.unshift(File.dirname(__FILE__))
-$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'rails'))
-require "rails-settings-cached"
-
-require "rails/railtie"
+require_relative '../lib/rails-settings-cached'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.